page-giving-rides.php

```diff
@@ -0,0 +1,50 @@
+<?php 
+  get_header(); 
+?>
+
+<!doctype html>
+
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+
+    <title>Giving Rides</title>
+    <meta name="Giving-Rides" content="Giving-Rides">
+    <link rel="stylesheet" href="giving-rides.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <div class="blue-box">
+      <div class="intro-text">Independence Rides is entirely volunteer-driven.</div>
+      <div class="intro-vertline"></div>
+      <div class="intro-text"> All drivers are courteous and trained volunteers who have passed criminal background and driving record checks. </div>
+    </div>
+    <div class="important-info">Important Information</div>
+    <ul id="info-list">
```

Comment:
note the comment in the giving rides css file that gives insight on changing this to a class